src/main/java/uk/gov/hmcts/reform/pip/account/management/model/MediaApplicationDto.java

```diff
@@ -11,20 +16,30 @@
 @AllArgsConstructor
 @NoArgsConstructor
 public class MediaApplicationDto {
+
+    @NotBlank(message = "fullName shouldn't be blank or null")
     private String fullName;
 
+    @Email
+    @NotBlank(message = "email shouldn't be blank or null")
     private String email;
 
+    @NotBlank(message = "employer shouldn't be blank or null")
     private String employer;
 
-    private MediaApplicationStatus status;
+    @NotNull(message = "status shouldn't be null")
+    @ValueOfEnum(
+        enumClass = MediaApplicationStatus.class,
+        message = "status should be one of PENDING, REJECTED or APPROVED"
+    )
+    private String status;
```

Comment:
Nothing wrong with this and no need to change this since it is done this way. There is another way to keep the Enum value and do validation with that. We can create a new method in the Enum class annotated with @JsonCreator so we can convert Enum and determine the value we received from the controller is expected. If not expected, throw a custom exception then handle the exception in GlobalExceptionHandler. This way is ok too. It is just an alternative if we want to stick with Enum in the Dto. Will approve the PR.

@JsonCreator
    public static MediaApplicationStatus create(String value) {
        if (value == null) {
            throw new IllegalArgumentException();
        }
        for (MediaApplicationStatus status : values()) {
            if (value.equals(status.toString())) {
                return status;
            }
        }
        throw new EnumConversionException();
    }

Reply:
Left it how it is for now just because the annotation I have added can be re-used for other models validations without changes. Like it though :)